ConsoleApp1Core/BaseCalculator.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApp1Core
+{
+	class BaseCalculator
+	{
+		int Add(int value1, int value2)
+		{
+			return value1 + value2;
+		}
+
+		int Substruct(int value1, int value2)
```

Comment:
Jak klikniesz na nazwę metody i tą żaróweczkę i wybierzesz opcje expression metod to będziesz mógł jąjeszcze bardziej skrócić